### ISSUE TITLE
Flow of narrative and examples from layout to collections

### ISSUE THREAD

**Issue by anenadic** [carpentries-incubator/jekyll-pages-novice#208]
This is a bigger issue to be split into sub-issues, recording it here while ti is still fresh:
- we should start with defining the home page layout and call it homepage-layout.html rather than default (learners found it confusing) - related to #209 
- make sure we have footer.html include defined ahead of this and that the text is italicised and contains the word "footer" to make it visually distinct
- define regular-page.html layout next to include contact details or other better example of extending should be thought of - this may be a bit too contrived as contact info could be on home page too - related to #209 too

**Comment by anenadic** [comment]
For footer.html use e.g.:
```
<footer>
  <em>Footer: an example website made with Jekyll.</em>
</footer>
```

**Comment by anenadic** [comment]
For homepage-layout.html use e.g.:
```
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>{{ page.title }}</title>
  </head>
  <body>
    {% include banner.html %}
    <h1>{{ page.title }}</h1>
      {{ content }}
     <footer>
         <em>Footer: an example website made with Jekyll.</em>
     </footer>
  </body>
</html>
```

Follow it up with an exercise to extract footer into an include.

**Comment by tobyhodges** [comment]
Closed via #242
